src/main/java/org/sagebionetworks/bridge/dao/ParticipantDataDao.java

```diff
@@ -0,0 +1,58 @@
+package org.sagebionetworks.bridge.dao;
+
+import org.sagebionetworks.bridge.models.ForwardCursorPagedResourceList;
+import org.sagebionetworks.bridge.models.ParticipantData;
+
+public interface ParticipantDataDao {
+
+    /**
+     * Get participant data records for the given userId
+     * @param userId - the userId for the participant data
+     * @return list of participant data records in a resource list that includes original query values
+     */
+    ForwardCursorPagedResourceList<? extends ParticipantData> getParticipantData(String userId, String offsetKey, int pageSize);
+
+    /**
+     * Get participant data for the given userId
+     * @param userId
+     * @param offsetKey
+     * @param pageSize
+     * @return
+     */
+    ForwardCursorPagedResourceList<ParticipantData> getParticipantDataV4(String userId, String offsetKey, int pageSize);
+
+    //TODO check the functions below..
+
+    /**
+     * Get participant data record for the given userId and configId
+     * @param userId
+     * @param configId
```

Comment:
configId for something named data? What about just identifier, or key?

Reply:
Do you mean that I should change configId to identifier? This would make sense because I think this was originally ParticipantConfig, but we changed it to ParticipantData and I didn't change configId. Oops, I'll go ahead and change it to identifier.